server/default-config.json

```diff
@@ -0,0 +1,6 @@
+{

```

Comment:
What is this file for?

Reply:
needed for mongo-migrate to specify url

Reply:
Then what is `server/migrations/default-config.json` for?

Reply:
oh shit nvm server/default-config.json isn't needed, didn't notice that